src/modules/Layout/Header/Header.jsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 
+import block from '../../../helpers/bem-cn';
 import Container from '../Container/Container';
-import JanusLogo from '../../UI/icons/JanusLogo';
 import Nav from '../Nav/Nav';
 
 import './Header.css';
 
+const b = block('j-header');
+
 const Header = ({ children }) => {
   return (
-    <header className="j-header">
-      <div className="j-header__col j-header__left">
-        <JanusLogo />
+    <header className={b}>
+      <div className={b('col', { 'left': true })}>
```

Comment:
@nik-kor it is like this `j-header__col--left`. It's modifier for column of the header, but not of the header itself